### PR TITLE
[stable/graylog] add apiVersion

### PR DESCRIPTION
**Pull request by cpanato**
Adding the `apiVersion` for `chart.yaml`

to fix the issue: https://github.com/helm/charts/issues/13763

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

File: stable/graylog/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
+apiVersion: v1
 name: graylog
 home: https://www.graylog.org
-version: 0.1.4
+version: 1.0.0
```
Comment (mattfarina) [inline stable/graylog/Chart.yaml:4]: Shouldn't this be `0.1.5`?

Comment (cpanato) [reply stable/graylog/Chart.yaml:4]: we agreed some time ago that all charts in the stable folder should be at least 1.0.0 @mattfarina

**Pull request merged**